### ISSUE TITLE
Is it possible to make a Mode called "VeryPoor"

### ISSUE THREAD

**Issue by franva** [kolton/d2bot-with-kolbot#505]
Once the VeryPoor mode is turned on, it picks up everything and sell them to save money?

rather than what I do right now is, set the gold >= 1, change several files here and there.

**Comment by rhoxthebeast** [comment]
There is a lowgold option in char config that sets a gold limit, below that
limit the bot will auto pick stuff to sell.

**Comment by winston-fan-webjet** [comment]
thanks man, could you be more specific please?

**Comment by fwaechter** [comment]
You don't have to change it in the Town.js. If you change it there it's true for all profiles. Just add `Config.LowGold = 150000` to your character config.

**Comment by DarkHorseDre** [comment]
Yeah thats what i meant, as in it is referenced in town.js but was not present in the configs, so I pasted it in
